### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Aniketks29**
Fixed typos

### CODE REVIEW

**Review by rthadur — APPROVED** [review]

**Comment by rthadur** [comment]
cc @mihaimaruseac

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Before sending your pull requests, make sure you followed this list.
 
 ### Contributor License Agreements
 
-We'd love to accept your patches! Before we can take them, we have to jump a couple of legal hurdles.
+We'd love to accept your patches! Before we can take them, you have to jump a couple of legal hurdles.
```
Comment (mihaimaruseac) [inline CONTRIBUTING.md:18]: This is "we" as in Google

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ just getting started, Github has a
 [how to](https://help.github.com/articles/using-pull-requests/).
 
 TensorFlow team members will be assigned to review your pull requests. Once the
-pull requests are approved and pass continuous integration checks, a TensorFlow
+pull requests are approved and passes continuous integration checks, a TensorFlow
```
Comment (mihaimaruseac) [inline CONTRIBUTING.md:36]: The correct form is "pass", since we're talking about multiple pull requests

**Review by mihaimaruseac — COMMENTED** [review]
This should be reverted

**Comment by rthadur** [comment]
@mihaimaruseac thank you

**Pull request merged**